src/graphql/entries.ts

```diff
@@ -83,19 +85,76 @@ function getContentTypenameFromEntityReferenceMap(
   if (referenceMap && entityId) {
     const entity = referenceMap.get(entityId);
     if (entity) {
-      const contentTypeId = entity.sys.contentType?.sys.id;
+      const contentTypeId = entity.sys.contentType.sys.id;
       const typename = contentTypeId.charAt(0).toUpperCase() + contentTypeId.slice(1);
       return typename;
     }
   }
 }
 
+const ASSET_TYPENAME = 'Asset';
```

Comment:
This to me feels like it could live in a `graphql/types.ts` file

Reply:
added it behind a debug flat, that the customer can enable 👍